### ISSUE TITLE
Index Error

### ISSUE THREAD

**Issue by ONSKJ** [TilCreator/Tapas-Comic-Downloader#14]
Hello!

It's me again! I don't know if this is only a problem on my type of computer (Windows 8) but when  tried using your new code, I received an error for every single input I used. Before the code worked on any comic, just not the ones where I encountered unicode support errors. But now, I get an Index Error before the code can even fully execute.  This is the error I get: 

![Capture](https://user-images.githubusercontent.com/60533679/75932163-c4c12200-5e44-11ea-8ebc-3d1d4ac57b92.PNG)

**Comment by TilCreator** [comment]
Tapas seams to have changed the site, so my current method doesn't work anymore.
Best thing to do would be to monitor the network traffic of the app and use the API the app uses. But I sadly will not have time for the next months.

So if someone can find a way to make the old method work again, the script will be fixed quickly, but else it sadly will be a few months.

See the extra section in the README.md: The _data variable is not used anymore, so I can't use it like usual for getting all the page ids.

**Comment by TilCreator** [comment]
Hmm, I can find direct links to the next page on ever page, maybe that could be used, will have a look in the next few days.

**Comment by TilCreator** [comment]
Notes:
- series page first episode example: (https://tapas.io/series/erma)
``` html
<a class="additional__btn additional__btn--quince" href="/episode/255222">Read Ep.1 </a>
```
- episode page: (view-source:https://tapas.io/episode/255222)
``` html
<a class="tab__button--small js-next-ep-btn ga-tracking" href="/episode/257351"
data-ga-category="Episode"
data-ga-action="Click"
data-ga-label="episode_w_tab"
data-permalink="https://tapas.io/episode/255222"
data-where="Episode"
data-id="257351"><i class="ico--arrow sp-ico-arrow-next"></i>Next</a>
```

**Comment by l1m3r** [comment]
> 
> 
> Hmm, I can find direct links to the next page on ever page, maybe that could be used, will have a look in the next few days.

sorry for this little shameless self plug, but I just updated [my tapas.io downloader script](https://github.com/l1m3r/tapas.io-phpDLer) to work with the new website.

feel free to copy the new regexes.

**Comment by TilCreator** [comment]
Have you found a new list of page ids or is it just he next links?

Also thx for the offer, but I'm not parsing xml with regex (again) ^^

**Comment by l1m3r** [comment]
I never worked with a/the list of page IDs -> The script always only use the next page links.
I didn't even modify your old script to work with the name of the comic - it still needs a episode number to start/work with.  :-/

**Comment by TilCreator** [comment]
I think my rewrite was worth it, still meh, that it broke.

I will hopefully have time to fix the tomorrow.

**Comment by l1m3r** [comment]
well just today i wanted to use my script again and it didn't work anymore. Guess what my first step was: _Hmmm, let's see if TilCreators new script still works or was updated to work recently._ ;-)

If your new one had worked now I would've had the choice between A) updating mine and B) installing python on my Windows to run yours -> might have chosen B and abandoned mine...

**Comment by TilCreator** [comment]
Fixed in: 4a524501fbccd8a3f102282b23cdfa9b1cb62dae
